node_modules/oae-util/lib/server.js

```diff
@@ -86,6 +86,16 @@ var setupServer = module.exports.setupServer = function(port, _config) {
         return next();
     });
 
+    // Trim whitespace on all post parameters
+    app.use(function(req, res, next) {
+        _.each(req.body, function(value, key) {
+            if (_.isString(value)){
+                req.body[key] = value.trim();
+            }
+        });
+        return next();
+    });
```

Comment:
Agreed. Lets just take one or a few of our endpoints and feed it some data with trailing spaces (including spaces only strings) and verify that it processes them as expected